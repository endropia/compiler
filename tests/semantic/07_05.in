function first(): integer;
	begin
	end;

function first(): integer;
	begin
	end;

begin
end.