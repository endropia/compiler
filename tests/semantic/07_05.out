Id is already declared in scope